README.md

```diff
@@ -73,22 +80,36 @@ dependencies. Next run `yarn start` to start storybook!
 Something you want to see your changes immediately in your project. Follow these steps to link asc
 to your project.
 
-1. Run `yarn build:watch`, this will watch your files and transpile them to the package lib 
+1. Run `yarn build:watch`, this will watch your files and transpile them to the package lib
    directory.
 2. cd to the package you want to test, run `yarn link`.
-3. Go to the repo where you want to use your package and run 
-   `yarn link <package-name>`. The `<package-name>` can be found in the `package.json` you linked in 
+3. Go to the repo where you want to use your package and run
+   `yarn link <package-name>`. The `<package-name>` can be found in the `package.json` you linked in
    step 2.
 
 Now you can import the package like you would do like a normal npm dependency. Changes you will make 
 in your package will be seen in your repo.
 
-- There is a known issue when developing with `yarn link` and using Hooks 
-  [click here for details](https://reactjs.org/warnings/invalid-hook-call-warning.html). The cause 
+- There is a known issue when developing with `yarn link` and using Hooks
```

Comment:
Does this really still occur?

Reply:
yes on my dev computer, I let it here for the moment